helios-client/src/main/java/com/spotify/helios/common/Resolver.java

```diff
@@ -44,7 +44,8 @@
 
 public abstract class Resolver {
 
-  private static final String SRV_FORMAT = env("HELIOS_SRV_FORMAT", "_%s._http.%s");
```

Comment:
Making a new env var to not break users, if any, that were using `HELIOS_SRV_FORMAT`.